GliaWidgets/SecureConversations/SecureConversations.ChatWithTranscriptModel.swift

```diff
@@ -358,6 +364,33 @@ extension SecureConversations {
                 return
             }
         }
+
+        @discardableResult
+        private func validateMessage() -> Bool {
```

Comment:
i don't like this method because of the name, it says an action but returns a boolean. And because it requires the messageText property instead of asking for it as a parameter. But I also understand that this comes from the ChatViewModel, so maybe we should just keep it like that.

Reply:
Yes, it is taken from chat implementation. And because it is not data-driven, it does some side effects  along with returning value. Since for now we are tied to ChatView, I think we have to be consistent between models.